lib/please_cli/please_cli/deploy.py

```diff
@@ -463,3 +463,117 @@ def cmd_TASKCLUSTER_HOOK(ctx,
         output,
         ask_for_details=interactive,
     )
+
+
+@click.command()
+@cli_common.cli.taskcluster_options
+@click.argument(
+    'project',
+    required=True,
+    type=click.Choice(please_cli.config.PROJECTS),
+    )
+@click.option(
+    '--channel',
+    type=click.Choice(please_cli.config.CHANNELS),
+    envvar="GITHUB_BRANCH",
+    required=True,
+    )
+@click.option(
+    '--nix-build',
+    required=True,
+    default=please_cli.config.NIX_BIN_DIR + 'nix-build',
+    help='`nix-build` command',
+    )
+@click.option(
+    '--nix',
+    required=True,
+    default=please_cli.config.NIX_BIN_DIR + 'nix',
+    help='`nix` command',
+    )
+@click.option(
+    '--docker-repo',
+    required=True,
+    default=please_cli.config.DOCKER_REPO,
+    help='Docker repository.',
+    )
+@click.option(
+    '--image-tag-format',
+    default='{project}-{channel}',
+    help='Docker image tag format. Accepted templates: {project}, {channel}',
+    )
+@click.option(
+    '--interactive/--no-interactive',
+    default=True,
+    )
+@click.pass_context
+def cmd_DOCKERHUB(ctx,
+                  project,
+                  channel,
+                  nix_build,
+                  nix,
+                  taskcluster_secret,
+                  taskcluster_client_id,
+                  taskcluster_access_token,
+                  docker_repo,
+                  image_tag_format,
+                  interactive,
+                 ):
+    '''Push to Docker Hub.
+
+    Creates versioned ($project-$hash) and stable (*-$channel) tags.
+    '''
+
+    secrets = cli_common.taskcluster.get_secrets(
+        taskcluster_secret,
+        project,
+        required=(
+            'DOCKER_USERNAME',
+            'DOCKER_PASSWORD',
+        ),
+        taskcluster_client_id=taskcluster_client_id,
+        taskcluster_access_token=taskcluster_access_token,
+    )
+
+    docker_username = secrets['DOCKER_USERNAME']
```

Comment:
I would make `docker_username` an command option that would default to `secrets['DOCKER_USERNAME']` and I would do the same for `docker_password`